fourstoryFlask/routes.py

```diff
@@ -59,8 +55,7 @@ def authenticate():
     return redirect(f'/history/date/{today}')
 
 @bp.route('/history/date/<date>')
-def history(date):
-    date_str = request.view_args['date']
```

Comment:
One other tiny clean-up: the `view_args` are the same thing that's passed into the view function